blogs/2016/03/11/ExtensionsRoundup.md

```diff
@@ -10,21 +10,21 @@ Author: Wade Anderson
 
 # VS Code Extensions
 
-Three months have passed since we open sourced VS Code and introduced the [extensions API](http://code.visualstudio.com/docs/extensionAPI/vscode-api). 
-The [Extensions Marketplace](https://marketplace.visualstudio.com/VSCode) has over 850 extensions. Many new languages and frameworks are now supported.
+In November 2015 we open sourced VS Code and introduced the [extensions API](http://code.visualstudio.com/docs/extensionAPI/vscode-api). 
+The VS Code [extensions Marketplace](https://marketplace.visualstudio.com/VSCode) has over 850 extensions. Many new languages (Go, PowerShell, Python) and frameworks (Apache Cordova, Chrome browser) are now supported.
```

Comment:
After I sent this, I thought PHP might be better to have in the list instead of PowerShell.  Or maybe PHP counts as a framework and it could go in that list?